### DISCUSSION TITLE
Cyber Security Approval for DAX Studio

### DISCUSSION THREAD

**Discussion by jkenne01** [DaxStudio/DaxStudio#939] [General]
Hi,

Next week I am taking DAX Studio  for approval at my company along with Tabular Editor.

As we are a Bank, any knew tool is closely scrutinised by Cyber Security.

I have received this from the Tabular Editor team : https://docs.tabulareditor.com/te3/features/security-privacy.html

is there any such similar content for DAX Studio?  Or would the same apply? Thanks in advance

**Comment by dgosbell** [comment]
DAX Studio is very similar to TE from an architecture perspective. I could probably use the TE content as a starting point and build out a similar page for DAX Studio. It's probably a good thing to have as I'm sure other organizations have similar questions.

**Reply by dgosbell** [reply]
I've just published a page that covers the same sort of information as the Tabular Editor one here https://daxstudio.org/docs/security-privacy/

**Comment by jkenne01** [comment]
Hi Darren,

This is really great. Thanks so much. I am at Bank of Queensland and
presenting Tabular Editor3, Dax Studio and ALM toolkit for cyber approval
next week. This will be very useful to include in the presentation.

Thanks for all of your amazing  efforts with this tool.

Cheers,
Jack

**Comment by jkenne01** [comment]
Hi Darren,

We are slowly progressing with our approval for Dax studio. I wonder do you
have a contact email so that our cyber security team can send you a
questionnaire which is a requirement for all open source software approvals?

Thanks for your help!

Cheers,
Jack
